docpage/docs/02.-Inputs/dcache.md

```diff
@@ -1,7 +1,9 @@
+---
+sidebar_position: 2
+---
 # dCache
-The main goal of this ProcessGroup is to get the events from dcache.
 
-It is necessary:
+dCache is the ProcessGroup that keep listen into dCache instance, is comprised of:
```

Comment:
How about?: "dCache is the ProcessGroup that monitors dCache instances for events"